static-build/test/exports.test.lua

```diff
@@ -81,6 +81,7 @@ local check_symbols = {
     'space_run_triggers',
     'space_bsize',
     'box_schema_version',
+    'box_select_ffi',
```

Comment:
This function isn't public so it shouldn't be checked. Please remove.

Reply:
But there is plenty of function that aren't part of public API. I see that all symbols are checked, even ones that needed for FFI only.

Reply:
I assume they're checked because they are used by some tdg modules (static build was initially imported from tarantool-enterprise). Apparently, `box_select_ffi` can't be used by any modules, and we don't want it to ever be used so we don't need to check it.

Reply:
OK, removed it